VideoLocker/src/androidTest/java/org/edx/mobile/test/http/MockApiTests.java

```diff
@@ -0,0 +1,209 @@
+package org.edx.mobile.test.http;
+
+import android.text.TextUtils;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.squareup.okhttp.mockwebserver.Dispatcher;
+import com.squareup.okhttp.mockwebserver.MockResponse;
+import com.squareup.okhttp.mockwebserver.MockWebServer;
+import com.squareup.okhttp.mockwebserver.RecordedRequest;
+
+import org.edx.mobile.util.Config;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Random;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * This class will run all the tests defined in the ApiTests class on a mock
+ * server
+ */
+public class MockApiTests extends ApiTests {
```

Comment:
Our goal is to get rid of ApiTest which uses real http request. 
So please feel free to modify ApiTests code itself.  
otherwise testing code of ApiTests will still be executed.